docs/docs/configuration/reference.md

```diff
@@ -656,6 +657,8 @@ cameras:
     # If disabled: config is used but no live stream and no capture etc.
     # Events/Recordings are still viewable.
     enabled: True
+    # Optional: camera type used for some Frigate features (default: shown below)
```

Comment:
probably good to add a NOTE: with possible options, similar to some other enum fields

Reply:
Right, good point